data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -1,5 +1,406 @@
+                [/has_attack]
+            [/filter]
+
+            [set_variable]
+                name = {DISTINGUISHER}_used
+                value = 0
+            [/set_variable]
+        [/modify_unit]
+    [/event]
+[+attack]
+[+specials]
+# wmlxgettext: [/specials]
+# wmlxgettext: [/attack]
+#enddef
+
+#define WEAPON_SPECIAL_NOVA
+# wmlxgettext: [attack]
+# wmlxgettext: [specials]
+    [dummy]
+        id = nova
+        name = _"nova"
+        description= _ "This weapon, when used offensively, deals damage to all units adjacent to the caster when it hits."
+        special_note=_ "This unit has wide-area attacks centered on the caster."
+        active_on = offense
+    [/dummy]
+[/specials]
+[/attack]
+    [event]
+        name = attacker_hits
+        id = nova
+        first_time_only = no
+        [filter_attack]
+            special_id = nova
+        [/filter_attack]
+        [harm_unit]
+            [filter]
+                [not]
+                    find_in = second_unit
+                [/not]
+                [filter_adjacent]
+                    find_in = unit
+                [/filter_adjacent]
+            [/filter]
+            amount = $weapon.damage
+            damage_type = $weapon.type
+            alignment = $unit.alignment
+            kill = yes
+            experience = yes
+            fire_event = yes
+        [/harm_unit]
+    [/event]
+[+attack]
+[+specials]
+# wmlxgettext: [/specials]
+# wmlxgettext: [/attack]
+#enddef
+
+#define WEAPON_SPECIAL_RAY
+# wmlxgettext: [attack]
+# wmlxgettext: [specials]
+    [dummy]
+        id = ray
+        name = _"ray"
+        description= _ "This weapon, when used offensively, also deals damage to a unit behind the target."
+        active_on = offense
+    [/dummy]
+[/specials]
+[/attack]
+    [event]
+        name = attacker_hits
+        id = ray_hits
+        first_time_only = no
+        [filter_attack]
+            special_id = ray
+        [/filter_attack]
+        [harm_unit]
+            [filter]
+                [filter_adjacent]
+                    find_in = second_unit
+                    formula = "
+                        unit_at(direction_from(self.loc,caster.facing)) = other
+                    where
+                        caster = unit_at(direction_from(self.loc, self.facing))
+                    "
+                [/filter_adjacent]
+            [/filter]
+            amount = $weapon.damage
+            damage_type = $weapon.type
+            alignment = $unit.alignment
+            kill = yes
+            experience = yes
+            fire_event = yes
+        [/harm_unit]
+    [/event]
+[+attack]
+[+specials]
+# wmlxgettext: [/specials]
+# wmlxgettext: [/attack]
+#enddef
+
+#define ABILITY_TAILWIND X Y
+    [dummy]
+        id = tailwind_{X}
+        name = _ "tailwind"+{X}
+        description = _ "At the start of the turn this unit increases movement points of surrounding units by " + "{X}"
```

Comment:
This also seems like it would have translation problems of the same sort as the other two strings I've commented on. The solution is probably the same.